<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ecosongs</class>
 <widget class="QMainWindow" name="Ecosongs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>657</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="PageStackedWidget" name="pages">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>6</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="DBExplorer" name="db_page">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="AudioManager" name="audio_page"/>
      <widget class="Analysis" name="analysis_page"/>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="sidebar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>93</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">#sidebar{
  	show-decoration-selected: 1; 
	outline:none;
}
#sidebar::item {
	border: 1 outset rgb(136, 138, 133);
	border-radius: 2;
	background-color: rgb(238, 238, 236);
	padding: 5 5 5 5;
	margin:0 2 2 2;
	width:75;
/*background-color:blue;*/
}


#sidebar::item:selected:!active {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #ABAFE5, stop: 1 #8588B2);
}

#sidebar::item:selected:active {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #6a6ea9, stop: 1 #888dd9);
	selection-background-color: green;
}

#sidebar::item:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #FAFBFE, stop: 1 #DCDEF1);
}

#sidebar::text:selected {
    border:none;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
      <property name="flow">
       <enum>QListView::TopToBottom</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::IconMode</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="currentRow">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>Database Manager</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
       <property name="icon">
        <iconset resource="resources/ecosongs.qrc">
         <normaloff>:/tango/db-manager</normaloff>:/tango/db-manager</iconset>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Audio</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
       <property name="icon">
        <iconset resource="resources/ecosongs.qrc">
         <normaloff>:/tango/audio</normaloff>:/tango/audio</iconset>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Analysis</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
       <property name="icon">
        <iconset resource="resources/ecosongs.qrc">
         <normaloff>:/tango/analysis</normaloff>:/tango/analysis</iconset>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1057</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="aOpen"/>
    <addaction name="separator"/>
    <addaction name="aOpen_db"/>
    <addaction name="aSave_db"/>
    <addaction name="aExport_db"/>
    <addaction name="separator"/>
    <addaction name="aExit"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="aHelp"/>
    <addaction name="separator"/>
    <addaction name="aAbout"/>
   </widget>
   <widget class="QMenu" name="menu_tools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="wac2wav"/>
    <addaction name="aSettings"/>
   </widget>
   <widget class="QMenu" name="menu_analyses">
    <property name="title">
     <string>Analyses</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_recording">
    <property name="title">
     <string>Recording</string>
    </property>
    <widget class="QMenu" name="menu_calculate">
     <property name="title">
      <string>Calculate...</string>
     </property>
     <addaction name="action_ACI"/>
    </widget>
    <addaction name="menu_calculate"/>
    <addaction name="separator"/>
    <addaction name="action_2wav"/>
    <addaction name="action_details"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_analyses"/>
   <addaction name="menu_recording"/>
   <addaction name="menu_tools"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_import_files"/>
   <addaction name="aSpecScale"/>
   <addaction name="action_highlight_predictions"/>
  </widget>
  <action name="aAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="aHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="aOpen">
   <property name="text">
    <string>Open File(s)</string>
   </property>
  </action>
  <action name="aExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="wac2wav">
   <property name="text">
    <string>Convert WAC files</string>
   </property>
  </action>
  <action name="aOpen_db">
   <property name="text">
    <string>Load database</string>
   </property>
  </action>
  <action name="aSave_db">
   <property name="text">
    <string>Save database</string>
   </property>
  </action>
  <action name="aExport_db">
   <property name="text">
    <string>Export database</string>
   </property>
  </action>
  <action name="action_ACI">
   <property name="text">
    <string>Acoustic Complexity Index (ACI)</string>
   </property>
  </action>
  <action name="action_details">
   <property name="text">
    <string>Show Details</string>
   </property>
  </action>
  <action name="action_2wav">
   <property name="text">
    <string>Convert to wav</string>
   </property>
  </action>
  <action name="aSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="aSpecScale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Select spectrogram scale</string>
   </property>
  </action>
  <action name="action_import_files">
   <property name="icon">
    <iconset resource="resources/ecosongs.qrc">
     <normaloff>:/tango/file-new</normaloff>:/tango/file-new</iconset>
   </property>
   <property name="text">
    <string>Import audio files</string>
   </property>
   <property name="toolTip">
    <string>Import audio files</string>
   </property>
  </action>
  <action name="action_highlight_predictions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Highlight files with predictions</string>
   </property>
   <property name="toolTip">
    <string>Highlit files for which predictions have been calculated</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DBExplorer</class>
   <extends>QWidget</extends>
   <header>gui.widgets.dbmanager.dbexplorer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AudioManager</class>
   <extends>QWidget</extends>
   <header>gui.widgets.audio.audiomanager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Analysis</class>
   <extends>QWidget</extends>
   <header>gui.widgets.analysis.analysis.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PageStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>gui.widgets.common.page_stacked_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/ecosongs.qrc"/>
 </resources>
 <connections/>
</ui>
